### DISCUSSION TITLE
New face!

### DISCUSSION THREAD

**Discussion by Odysseus707** [MDAnalysis/mdanalysis#4467] [GSoC Discussions]
Hi!

I'm **Vivek**, a third-year B.Tech student majoring in Computer Science from India. I've been diving deep into the world of Data Science through various projects and internships for the last ~1.5 years, so I'm pretty comfortable with it.

I've had some basic exposure to the MDAnalysis library in a short project in my sophomore year project under my professor. As such, I've got a solid background in Python and data science/ML libraries. I did organic chemistry till 12th grade and have a rudimentary understanding of biochemistry.

Lately, I've been browsing the guide to learn how to contribute. Just starting with GSoC 2024, I'm excited about the opportunity to apply what I'm learning and make meaningful contributions to MDAnalysis.

Exciting times ahead!

**Comment by hmacdope** [comment]
Hi Vivek! Welcome! Have a look at our contributing guide for more info:https://userguide.mdanalysis.org/stable/contributing.html
